src/commands/which.ts

```diff
@@ -3,6 +3,12 @@ import {Command, CliUx} from '@oclif/core'
 export default class Which extends Command {
   static description = 'Show which plugin a command is in.'
   static strict = false;
+  static examples = [
+    {
+      description: 'See which plugin the `help` command is in:',
+      command: '<%= config.bin %> <%= command.id %> help',
+    },
+  ]
 
   async run(): Promise<void> {
     const {argv} = await this.parse(Which)
```

Comment:
this used to be an oclif arg, but we parse `argv` to support space-separated commands without quotes, see:
https://github.com/oclif/plugin-which/pull/194